### PR TITLE
Save open tabs to list

### PR DESCRIPTION
**Pull request by sharvaniharan**
Save articles from tabs list to a reading list

**Phab**: https://phabricator.wikimedia.org/T224339

### CODE REVIEW

File: app/src/main/java/org/wikipedia/page/tabs/TabActivity.java [hunk 1]
```diff
@@ -278,6 +285,16 @@ public boolean onOptionsItemSelected(MenuItem item) {
         return super.onOptionsItemSelected(item);
     }
 
+    private void saveTabsToList() {
```
Comment (cooltey) [inline app/src/main/java/org/wikipedia/page/tabs/TabActivity.java]: Would it be better to add the logic of checking "empty" tab when saving to list?

Comment (sharvaniharan) [reply app/src/main/java/org/wikipedia/page/tabs/TabActivity.java]: good one will add..

Comment (sharvaniharan) [reply app/src/main/java/org/wikipedia/page/tabs/TabActivity.java]: Done @cooltey

Comment (cooltey) [reply app/src/main/java/org/wikipedia/page/tabs/TabActivity.java]: Nice! Can we have a message if the "Save all tabs" action actually does nothing?

Comment (sharvaniharan) [reply app/src/main/java/org/wikipedia/page/tabs/TabActivity.java]: have asked for a copy on ticket @cooltey .. if everything else is good we can merge and then add the copy as follow-up?

Comment (cooltey) [reply app/src/main/java/org/wikipedia/page/tabs/TabActivity.java]: Make sense!

**Pull request merged**